Gemfile

```diff
@@ -1,5 +1,8 @@
 source 'https://rubygems.org'
 
+# rails-assets requires >= 1.8.4
+gem 'bundler', '>= 1.8.4'
```

Comment:
Something like this in the Gemfile sounds about right:

if Gem::Version.new(Bundler::VERSION) < Gem::Version.new('1.8.4')
  abort "Bundler version >= 1.8.4 is required"
end